src/internal.c

```diff
@@ -5216,6 +5216,8 @@ static int Receive(WOLFSSL* ssl, byte* buf, word32 sz)
                             return WANT_READ;
                         }
                     }
+				#elif defined(FREERTOS_TCP)
+					return WANT_READ;	/* want read, call again    */
```

Comment:
Instead of checking for WOLFSSL_CBIO_ERR_ISR first, then returning WANT_READ or WANT_WRITE here, I think it would be cleaner to simply return WOLFSSL_CBIO_ERR_WANT_READ or WOLFSSL_CBIO_ERR_WANT_WRITE in your custom I/O callbacks when necessary.  Does this make sense?

These lines also use hard tabs.  wolfSSL coding standard uses spaces instead.